.asf.yaml

```diff
@@ -23,4 +23,7 @@ github:
   enabled_merge_buttons:
     squash:  true
     merge:   false
-    rebase:  false
\ No newline at end of file
+    rebase:  false
+
+publish:
+  whoami: master
```

Comment:
I have doubts about this branch setting. As for dubbo-website, Github Actions will automiaticlly compile and deploy static files to https://github.com/apache/dubbo-website/tree/asf-site branch.